master/getting-started/kubernetes/installation/config-options.md

```diff
@@ -69,10 +69,16 @@ be undone):
 - Optionally, (to save some resources if you're running a VXLAN-only cluster) completely disable Calico's BGP-based
   networking:
   - Replace `calico_backend: "bird"` with `calico_backend: "vxlan"`.  This disables BIRD.
-  - Comment out the line `- -bird-ready` from the calico/node readiness check (otherwise disabling BIRD will cause the
+  - Comment out the line `- -bird-ready` and `- -bird-live` from the calico/node readiness/liveness check (otherwise disabling BIRD will cause the
     readiness check to fail on every node):
```

Comment:
Already approved, but worth removing "readiness" here: it's both, but I don't think it needs restating.

Reply:
Done :)